### ISSUE TITLE
ImagePicker base64 becomes really large on Android

### ISSUE THREAD

**Issue by dylancompanjen** [expo/expo#2125]
I already posted this in the forums 25days ago, at first I got a reply but later it was closed due to inactivity. (https://forums.expo.io/t/imagepicker-base64-becomes-really-large-on-android/11931)

### Environment

Environment:
  OS: macOS Sierra 10.12.6
  Node: 9.11.2
  Yarn: Not Found
  npm: 5.6.0
  Watchman: 4.7.0
  Xcode: Xcode 9.0 Build version 9A235
  Android Studio: 2.2 AI-145.3537739

Packages: (wanted => installed)
  expo: ^27.0.0 => 27.0.1
  react: 16.3.1 => 16.3.1
  react-native: https://github.com/expo/react-native/archive/sdk-27.0.0.tar.gz => 0.55.2

Diagnostics report:
  https://exp-xde-diagnostics.s3.amazonaws.com/oneshoe-bbe64627-c8bb-42bc-b8c7-8c161a9b227b.tar.gz

### Issue

We let users upload photo’s by selecting them with the ImagePicker. When they submit it, the base64 output is sent to our API. Our API accepts a maximum size of 8mb and we noticed that on Android devices a lot of times this maximum size is exceeded.

I just did a test with a small image and saw that the base64 result of ImagePicker (image of 41kb) becomes 13kb for iOS and 79kb for Android. So the base64 version is almost double the size of the original on Android.

```
const result = await ImagePicker.launchImageLibraryAsync({
  mediaTypes: ImagePicker.MediaTypeOptions.Images,
  exif: true,
  base64: true
})

if (!result.cancelled) {
  result.uploading = false
  result.error = false
  console.log(result.base64)
}
```

This happens in production on SDK27 but I also noticed it in our dev env with SDK28.
I tested it on a Google Pixel 8.0 - API 26 simulator and our client noticed it with several Android devices (so I think this happens on all or most of them).

Our client experienced it on the following devices: Sony Xperia Z5 Compact, Nexus 6P and Samsung A3.
On the Nexus 6P 300 kB became 760kB for e.g.

**Comment by m4tx** [comment]
This issue was addressed by commit 7548f07cf956d88e15176d3512dd98a83448da6c (related issue: #1970). Please upgrade your expo SDK version to 30.x and the problem should be gone.

**Comment by sjchmiela** [comment]
I'll close this issue tentatively, please submit a new issue if it happens again! 🙂
